### PR TITLE
able to show chatscreen

### PR DESCRIPTION
**Pull request by PrStWn**
Description, Motivation and Context
Create [MOBILE] CONSTRUCT UI CHAT SCREEN and [MOBILE] CREATE WIDGET BUBBLE CHAT

Link to the resources (mockups, change description)
https://trello.com/c/48m42mVP (this link for construc ui chatscreen),  and this link for Widget bubble chat https://trello.com/c/czaLfopm .

What is the destination branch of this PR?
development
How Has This Been Tested?
Screenshots [Mandatory]
image
image

### CODE REVIEW

File: lib/screens/chat_screen.dart [hunk 1]
```diff
@@ -20,8 +84,82 @@ class ChatScreen extends StatelessWidget {
...
       ),
-      body: Center(
-        child: Text('This is Chat Screen for $title'),
+      body: Column(
+        children: [
+          Expanded(
+            child: ListView.builder(
+                itemCount: chatBubbles.length,
+                itemBuilder: (context, index) {
+                  bool shouldDisplayName = index == 0 ||
```
Comment (josuastg) [inline lib/screens/chat_screen.dart]: ini tolong dibuat variable diatas ya @PrStWn mengurangi kompleksitas logic di widgetnya

Comment (PrStWn) [reply lib/screens/chat_screen.dart]: @josuastg yang dibuat variable yang mana ya? shouldDisplayName? atau yang mana

Comment (josuastg) [reply lib/screens/chat_screen.dart]: iya @PrStWn

File: lib/screens/chat_screen.dart [hunk 2]
```diff
@@ -20,8 +89,79 @@ class ChatScreen extends StatelessWidget {
...
+          Container(
+            height: 50,
+            width: MediaQuery.of(context).size.width,
+            child: Stack(
+              children: [
+                Align(
+                  alignment: Alignment.bottomCenter,
+                  child: Row(
+                    children: [
+                      Container(
```
Comment (josuastg) [inline lib/screens/chat_screen.dart]: ini juga ya @PrStWn 
image

**Review by josuastg — APPROVED** [review]

**Pull request merged**